app/src/main/res/values/strings.xml

```diff
@@ -9,4 +9,9 @@
     <string name="get_characters_error">Something went wrong, please try again</string>
     <string name="press_to_show_characters">Press to show Characters</string>
     <string name="name">name</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
same

Reply:
no, its not necessary jajajai removed it yet